File: sandbox.client/front/ts/model/ClientAddr.ts
import {AddrType} from "./AddrType";

export class ClientAddr {

    public client:number;
    public type: AddrType;
    public street: string;
    public house: string;
    public flat: string;

    constructor() {
        this.street = '';
        this.house = "";
        this.flat = '';
    }

    assign(o: any): ClientAddr {
        this.type = o.type;
        this.street = o.street;
        this.house = o.house;
        this.flat = o.flat;
        return this;
    }

}

File: sandbox.controller/src/kz/greetgo/sandbox/controller/model/model/ClientAddr.java
package kz.greetgo.sandbox.controller.model.model;

public class ClientAddr {
    public Integer client;
    public AddrType type;
    public String street;

    @Override
    public String toString() {

        return "ClientAddr{" +
          "client=" + client +
          ", type=" + type +
          ", street='" + street + '\'' +
          ", house='" + house + '\'' +
          ", flat='" + flat + '\'' +
          '}';
    }

    public String house;
    public String flat;
}
